addons/stock_calendar/resource.py

```diff
@@ -29,6 +30,7 @@ def _calculate_next_day(self, cr, uid, ids, fields, names, context=None):
         }
 
     # Keep as it takes into account times
+    @api.cr_uid_ids_context
```

Comment:
That's not correct. It's not how we do it. You should migrate all the method that override the original one, with an api.multi in this case